docs/source/user_guide/analysis.rst

```diff
@@ -122,14 +122,14 @@ method. By default the moment interaction diagram is generated for bending about
   :noindex:
 
 The moment interaction diagram is generated by shifting the neutral axis throughout the
-cross-section from pure bending to the decompression point using ``n_points``. A
-straight line is generated between the decompression point and the squash load, and the
-pure bending point and the tensile load.
+cross-section between the ``limits`` using either ``n_points`` or ``n_spacing``.
```

Comment:
wonder if this should be expended on a little to illustrate the difference between the two options

```suggestion
cross-section between the ``limits`` using either ``n_points`` number of equally spaced neutral axis depths or by determining neutral axis depths relating to ``n_spacing`` number of equally spaced axial load analysis points.
```

Reply:
Have moved this paragraph above the ``automethod``. Won't elaborate on ``n_points`` and ``n_spacing`` here as its covered in the docstring.